### PR TITLE
People can now witness a shuttle hijacker complete their hacking stage, rather than somehow be entirely oblivious to someone typing on a keyboard.

### PR DESCRIPTION
**Pull request by necromanceranne**
## About The Pull Request

Let's everyone know around the console who is attempting to hijack the shuttle once a stage is completed.

## Why It's Good For The Game

While the chaos of finding out who did it is fun, maybe you should try and cause a little more chaos in that cockpit before attempting to hijack. Currently you can just sorta get away with it a bit since nobody around you is informed about your hacking attempts. That seems entirely unrealistic and not exactly user friendly.

## Changelog

:cl:
balance: Sends a chat alert to those nearby when someone completes a stage of a hijack.
/:cl:

### CODE REVIEW

**Comment by exymian** [comment]
"[user.name] appears to be tampering with [src]."

is the message

**Comment by ATH1909** [comment]
shouldn't this be BEFORE the do_after(), since finishing a hacking stage already makes a big announcement that's broadcast to everyone?

**Comment by necromanceranne** [comment]
It does make a big announcement but not who did it. That's the issue to resolve for anyone nearby.

I would at it to the start, but I think this is fairer.

File: code/modules/shuttle/emergency.dm [hunk 1]
```diff
@@ -200,6 +200,9 @@
 /obj/machinery/computer/emergency_shuttle/proc/attempt_hijack_stage(mob/living/user)
 	if(!user.CanReach(src))
 		return
+	if(!HAS_TRAIT(user, TRAIT_HANDS_BLOCKED))
```
Comment (ShizCalev) [inline code/modules/shuttle/emergency.dm:203]: did you test this

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/modules/shuttle/emergency.dm (hunk 1 above)
Comment (necromanceranne) [reply code/modules/shuttle/emergency.dm:203]: Oops

**Pull request merged**